INTYGFV-15809: Spinner is not hidden after certificate gets deleted

SamuelTilly opened the pull request.

ChrilleSigerhed commented on apps/webcert/src/store/certificate/certificateMiddleware.ts:
```diff
@@ -241,12 +241,12 @@ const handleDeleteCertificateSuccess: Middleware<Dispatch> =
   ({ dispatch }) =>
   () =>
   (action: AnyAction): void => {
+    dispatch(hideSpinner())
```
Does it make more sense to move this dispatch to line 252, and hide the spinner after the other dispatches has finished?

SamuelTilly replied:
I don't think so, at least not in any meaningful way, it would be milliseconds for every dispatch to fire and update state

ChrilleSigerhed reviewed (approved):
See comment!

Also, would be great if we could add a test to verify that the spinner is indeed hidden correctly!

SamuelTilly commented:
> See comment!
> 
> Also, would be great if we could add a test to verify that the spinner is indeed hidden correctly!

Good point, added a test and moved the dispatch 👍
